File: dev/appscale-tests/tasks-test/primes.py
import sys
import math

print "Prime numbers"

#Asks user how many numbers he wants to calculate.
a = int(raw_input("Start?: "))
b = int(raw_input("End?: "))

if (a < 0) :
   print "Error: Invalid number. It must be greater than zero"
   sys.exit

if (a < 2) : # Discard 0 and 1 as primes
   a = 2     # Primes(0,10) == Primes(2,10)

count = 0 # number of already calculated primes.

for i in range(a,b) :
   
   for j in range(2,i) :
      if i % j ==0 :
         break
   else : # for's else; not if's else
      print i
      count += 1

print "Total: ", count

File: dev/appscale-tests/tasks-test/primes.c
#include<stdio.h>
#include<math.h>


int main()
{
   int i,j,a,b;
   int count;
   
   printf("Start?: ");
   scanf("%d",&a);
   
   printf("End?: ");
   scanf("%d",&b);
   
   for(i = a; i < b; i++)
   {
      for(j = 2; j < i; j++)
      {
         if(i % j == 0)
         break;
      }
      if(i == j)
      {
         count++;
         printf("\n%d",j);
      }
   }
   printf("\n");
   printf("Total: %i prime numbers\n",count);
   return 0;
}
